### PR TITLE
Add angreal 0.7.0 to conda-forge

### PR DESCRIPTION
**Pull request by sodre**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by sodre** [comment]
@conda-forge/help-python this recipe is ready for review

File: recipes/angreal/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - url: https://gitlab.com/angreal/angreal/-/raw/{{ version }}/requirements/dev.txt
+    sha256: 84b4610ee40438b7bf594944b6f148a9390de1473cbc074b71bbd9d23b3607ba
+  - url: https://gitlab.com/angreal/angreal/-/raw/{{ version }}/LICENSE.md
+    sha256: 589ed823e9a84c56feb95ac58e7cf384626b9cbf4fda2a907bc36e103de1bad2
+
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install . -vv
```
Comment (marcelotrevisani) [inline recipes/angreal/meta.yaml:25]:
```suggestion
  script: {{ PYTHON }} -m pip install . -vv
```

Comment (sodre) [reply recipes/angreal/meta.yaml]: Thank @marcelotrevisani! I forgot to change it back.

This should be good to go now.

**Comment by sodre** [comment]
@marcelotrevisani, any ideas why it would have failed? The only change was adding PYTHON back as you suggested.

File: recipes/angreal/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - angreal = angreal.cli:angreal_cmd
+
+
+requirements:
+  host:
+    - pip
```
Comment (marcelotrevisani) [inline recipes/angreal/meta.yaml:32]:
```suggestion
    - pip
    - python >=3.0
```

**Comment by marcelotrevisani** [comment]
> @marcelotrevisani, any ideas why it would have failed? The only change was adding PYTHON back as you suggested.

I think it is failing because is missing the ``python >3.0`` in ``host``

**Comment by marcelotrevisani** [comment]
@sodre I just add the ``python`` in ``host`` and it is passing now, should we integrate it then?

**Pull request merged**